### DISCUSSION TITLE
Add layout flag to customise logo/title style

### DISCUSSION THREAD

**Discussion by timothyhoward** [evidence-dev/evidence#2413] [General]
Feature Description
Recent versions of Evidence introduced the `logo` / `title` variables in the EvidenceDefaultLayout component. Could this potentially be further expanded to allow for style customisations without rebuilding the EvidenceDefaultLayout (with all subcomponents) as a custom layout? Primarily interested in increasing the height of the logo to allow for a bigger logo image (currently defaults to `h-5 aspect-auto`) and adding a background colour to the header.

Goal of Feature
Allow greater customisation of the layout and avoids having custom components that break with new releases.

Current Solution / Workarounds
At the moment I have to create custom EvidenceLayout, EvidenceHeader and Logo components. Would like to avoid having too many custom components if possible.

Alternatives
Allow passing through custom Tailwind style string for logo and header.

**Comment by earshinov** [comment]
I would also like the header to be more customizable. In my case I would like to add a GitLab icon to the header, but out of the box EvidenceDefaultLayout and the Header component have props only for GitHub, not GitLab. It would be fine by me if it was possible to pass a named slot to replace the <Header> within EvidenceDefaultLayout entirely.
